src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1698,4 +1698,16 @@
   <data name="ModuleInitializerAttributeShouldNotBeUsedInLibrariesTitle" xml:space="preserve">
     <value>The 'ModuleInitializer' attribute should not be used in libraries</value>
   </data>
+  <data name="UseAsyncMethodInAsyncContextMessage" xml:space="preserve">
+    <value>'{0}' synchronously blocks. Await '{1}' instead.</value>
+  </data>
+  <data name="UseAsyncMethodInAsyncContextTitle" xml:space="preserve">
+    <value>Call async methods when in an async method</value>
+  </data>
+  <data name="UseAsyncMethodInAsyncContextMessage_NoAlternative" xml:space="preserve">
+    <value>'{0}' synchronously blocks. Use await instead.</value>
+  </data>
+  <data name="UseAsyncMethodInAsyncContextDescription" xml:space="preserve">
+    <value>When inside a Task-returning method, use the async version of methods, if they exist.</value>
```

Comment:
The localization team will likely attempt to translate "Task" here.

Localization can be prevented like this:

https://github.com/dotnet/roslyn-analyzers/blob/affa88c580a9cffc5b016ad892c5bcd3971e8456/src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticsAnalyzersResources.resx#L378-L381